crates/turbopack-css/src/references/mod.rs

```diff
@@ -131,15 +131,20 @@ impl<'a> VisitAstPath for AssetReferencesVisitor<'a> {
         }
 
         let src = url_string(u);
-
-        self.references.push(
-            UrlAssetReferenceVc::new(
-                self.origin,
-                RequestVc::parse(Value::new(src.to_string().into())),
-                AstPathVc::cell(as_parent_path(ast_path)),
-            )
-            .into(),
-        );
+        // Ignore server-relative paths. These are currently not allowed in resolve.
+        //
+        // TODO: Create reference for server-relative paths and restore the original
+        // specifier during codegen?
+        if !src.starts_with('/') {
```

Comment:
Open to alternatives 😅

Reply:
What is the case here? Are we parsing the output from Google Fonts, or is this a file we're generating that we're using `/` absolute paths to reference other files?

Reply:
We download a stylesheet from Google fonts that references their fonts with absolute uris, but we then update it to reference the locally cached fonts instead: https://github.com/vercel/turbo/pull/2940/files#diff-89269fdf622b8700b55137d082919f2a22d65dae83a07dab0d90a846f2f45cc4R217

These lead with `/_next`, which we then trace and would otherwise create a reference for them, which isn't supported for server-relative paths.